src/main/java/org/folio/services/finance/transaction/PendingPaymentWorkflowService.java

```diff
@@ -137,11 +137,11 @@ private List<InvoiceWorkflowDataHolder> withNewPendingPayments(List<InvoiceWorkf
   }
 
   private Future<Void> updateTransactions(List<InvoiceWorkflowDataHolder> holders, RequestContext requestContext) {
-    var futures = holders.stream()
+    return GenericCompositeFuture.join(holders.stream()
       .map(InvoiceWorkflowDataHolder::getNewTransaction)
       .map(transaction -> baseTransactionService.updateTransaction(transaction, requestContext))
-      .collect(Collectors.toList());
-    return GenericCompositeFuture.join(futures).mapEmpty();
+      .collect(Collectors.toList()))
+      .mapEmpty();
   }
```

Comment:
@SinghAdes can you please insert link to the same implementation from mod-orders and check that there is no exceptions in the log exceptionally PENDING_PAYMENT_UPDATE_ERROR

Reply:
@SerhiiNosko :-  I have retested and there is no issue with the logs . Please find below link for mod-orders transaction update.

https://github.com/folio-org/mod-orders/blob/master/src/main/java/org/folio/service/finance/transaction/AbstractTransactionManagingService.java#:~:text=public%20Future%3CVoid,%7D

Reply:
[Invoice_logs.txt](https://github.com/folio-org/mod-invoice/files/10812450/Invoice_logs.txt)